statistics/ddl.go

```diff
@@ -45,48 +45,53 @@ func (h *Handle) DDLEventCh() chan *util.Event {
 
 // insertTableStats2KV inserts a record standing for a new table to stats_meta and inserts some records standing for the
 // new columns and indices which belong to this table.
-func (h *Handle) insertTableStats2KV(info *model.TableInfo) error {
+func (h *Handle) insertTableStats2KV(info *model.TableInfo) (err error) {
 	h.mu.Lock()
 	defer h.mu.Unlock()
 	exec := h.mu.ctx.(sqlexec.SQLExecutor)
-	_, err := exec.Execute(context.Background(), "begin")
+	_, err = exec.Execute(context.Background(), "begin")
 	if err != nil {
 		return errors.Trace(err)
 	}
+	defer func() {
+		err = doneTransaction(context.Background(), exec, err)
```

Comment:
What would be done to this `err`?

Reply:
`err` is a named return value, so it will be returned by the function after being assigned in the deferring part.